src/webextension/list/containers/item-filter.css

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.filter {
+  border: 1px solid #b6b6b6;
+  position: relative;
+  margin: 12px;
+}
+
+.filter:focus-within,
+.filter:focus-within:hover {
+  box-shadow: none;
+}
+
+.filter:before {
+  content: "";
+  display: block;
+  background-image: url(/icons/search.svg);
+  background-repeat: no-repeat;
+  background-size: 16px 16px;
+  height: 16px;
+  width: 16px;
+  position: absolute;
+  top: 7px;
+  left: 8px;
+}
+
+.filter input {
+  position: relative;
+  padding-left: 32px;
+}
+
+.filter button {
+  text-indent: -9999px;
```

Comment:
Instead of doing this to hide the text, we should remove the text, replace it with an image (either an `<img>` element or using the CSS below; however you think is best), and then give the button a title attribute for accessibility (in `src/webextension/widgets/filter-input.js`).

Reply:
@jimporter I will need some help in removing the "clear" copy, as I'm unsure how that gets pulled in using the widgets.ftl file.